app/views/pages/shared/_navbar.html.erb

```diff
@@ -0,0 +1,20 @@
+<nav>
+  <%= link_to root_path do %>
+    <%= image_tag "artify_logo", :height => "40", class: "logo"%>
+  <% end %>
+  <a href='#' data-item='Rent a Piece'>Rent a Piece</a>
+  <%# <a href='#' data-item='Collection'>Collection</a>
+  <a href='#' data-item='Top Rented'>Top Rented</a>
+  <a href='#' data-item='Contact-us'>Contact-us</a> %>
+  <%= render partial: "pages/shared/searchbar", class: "search-form"%>
```

Comment:
Again, `<%= render shared/searchbar %>`

But in truth your search form is only 4 lines long, there's really no need to add it in a partial. Can you add in this file directly and delete the searchbar partial ?